### DISCUSSION TITLE
Tabs and tab font is too large

### DISCUSSION THREAD

**Discussion by kenfhill84083** [freeplane/freeplane#2173] [Help]
As seen in the attached screenshot, the tabs and tab sizes are huge.  I used Command Search for a way to change it but nothing came up.  Can this be changed anywhere in Freeplane?  Thanks.

![freeplane _ tabs and tab font too large](https://github.com/user-attachments/assets/0bd40974-1792-4c33-a422-851075ba126d)

**Comment by euu2021** [comment]
`Preferences…->Appearance->Look and feel->Menu item font size`

**Reply by kenfhill84083** [reply]
Thanks that fixed the font size.  I did not explain the full problem.  The "X" and another symbol are still huge, forcing the tab to be huge.  Any fix for that?

![freeplane _ tabs and tab font too large 2](https://github.com/user-attachments/assets/1177c4fc-c173-45e5-9c0e-8e523cfdf060)

**Comment by dpolivaev** [comment]
Which Freeplane version are you using? How do your mind maps and icons appear? Could you also share what’s written in the log file? To assist you effectively, please provide as much detailed information as possible, even if some details might not seem relevant to you.

**Reply by kenfhill84083** [reply]
Freeplane  . 1.12.8    2024-12-07
I have been away from Freeplane for a while so I hope this is the right log file (from ...\AppData\Roaming\Freeplane\1.12.x\logs)

[log.zip](https://github.com/user-attachments/files/18261113/log.zip)

**Reply by dpolivaev** [reply]
Do you use multiple monitors? Have you moved Freeplane window between them? Can you upload a screenshot with a new map and zoom 100%?

**Reply by kenfhill84083** [reply]
Yes, I have 3 monitors and have used it on 2 that I remember (both 1080x1920).  
![freeplane _ laptop monitor](https://github.com/user-attachments/assets/ab5ffbb6-8ad9-4829-8fa6-499a152e7e88)
![freeplane _ landscape monitor](https://github.com/user-attachments/assets/fd615d83-541a-4d3d-bd30-ad7a85d2741f)
![freeplane _ portrait monitor](https://
[monitor test map.zip](https://github.com/user-attachments/files/18263017/monitor.test.map.zip)
github.com/user-attachments/assets/2469345d-a071-451d-a5ff-e2373360ac1b)

**Reply by dpolivaev** [reply]
I think I can fix it. Which value do you use for "Menu item font size"?

**Reply by kenfhill84083** [reply]
The screenshots show menu item font size of 6.  But later I changed it to 5.  I seldom use the menu because Command Search is so robust.

**Comment by euu2021** [comment]
Related issue: https://github.com/freeplane/freeplane/issues/1939

**Comment by dpolivaev** [comment]
Preview 1.12.9_01

**Reply by kenfhill84083** [reply]
much better.  thanks.
